lib/node_modules/@stdlib/math/base/special/roundsd/lib/native.js

```diff
@@ -0,0 +1,60 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var addon = require( './../src/addon.node' );
+
+
+// MAIN //
+
+/**
+* Rounds a numeric value to the nearest number with \\(n\\) significant figures.
+*
+* @private
+* @param {number} x - input value
+* @param {PositiveInteger} n - number of significant figures
+* @param {PositiveInteger} [b=10] - base
```

Comment:
```suggestion
* @param {PositiveInteger} b - base
```

Base won't be optional in the C implementation.